packages/react-scripts/template/README.md

```diff
@@ -176,6 +176,11 @@ If you aren’t satisfied with the build tool and configuration choices, you can
 
 Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
 
+Ejecting removes some automatic functionality like:
```

Comment:
I'm not very happy with this wording. Let's just mention this next to particular features. The way you did it below is fine.